opentok/opentok-linux-sdk-samples
# Introduce LIBOPENTOK_PATH environment variable

#### What is this PR doing?
This PR introduces the optional `LIBOPENTOK_PATH` environment variable to ease the process of building the samples with a local copy of the OpenTok SDK. This is specially useful in non-Debian Linuxes where no `libopentok-dev` package is available.

Currently, the CMake scripts are using `pkg-config` by default to search for an installation of the OpenTok SDK. Unfortunately, there is only a package for Debian, so this likely fails in non-Debian Linuxes, unless the user modifies the CMake scripts to point to a local copy of the SDK.

#### How should this be manually tested?
If you are on a Debian based Linux, make sure that you don't have the OpenTok SDK package installed (i.e. `sudo apt remove libopentok-dev`).

Download a copy of the SDK from https://tokbox.com/developer/sdks/linux/

Unpack it somewhere and set the LIBOPENTOK_SDK environment variable to point to the location where you extracted the SDK
```sh
export LIBOPENTOK_SDK=/home/opentok/sdk
```

Build any of the samples as you normally would.
